backend/routes/user.routes.ts

```diff
@@ -0,0 +1,59 @@
+import express from "express";
+const router = express.Router();
+const passport = require("passport");
+const bcrypt = require("bcrypt");
+const User = require("../models/User");
+
+import { DatabaseUserInterface } from "../utils/Interfaces";
+
+router.post("/signup", async (req, res) => {
+	const { username, password } = req?.body;
+	if (
+		!username ||
+		!password ||
+		typeof username !== "string" ||
+		typeof password !== "string"
```

Comment:
Why can't I not have a password of only numbers?
I actually think when you send it as a request, it is already a string because you're sending JSON (not 100% sure tho).